assets/js/vue/img.js

```diff
@@ -1,13 +1,23 @@
-// This is a very basic img component and doesn't have the logic that the ruby version has.
-// we need to build that logic to implement a somewhat same version of that component
-// https://github.com/nhoizey/jekyll-cloudinary/blob/master/lib/jekyll/cloudinary.rb
+import DefaultPresets from '../default-presets.js';
+import TransformationOptions from '../transformation-options.js';
+import Cloudinary from '../../../.storybook/config/cloudinary.js';
```

Comment:
Storybook existiert nicht im Scope von Jekyll, deshalb kannst du diese Inhalte auch nicht laden. Musst du umziehen. In Jekyll gibt es einen 404.